.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": [
+      "react-app",
+      "airbnb",
+      "plugin:jsx-a11y/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "plugins": [
+      "jsx-a11y"
+    ],
+    "rules": {
+      "semi": 1,
+      "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+      "no-unused-vars": 1,
+      // "no-multiple-empty-lines": 2,
+      "class-methods-use-this": 0,
+      "comma-dangle": ["error", "never"],
+      // "no-console": ["error", { "allow": ["warn", "log"] }],
+      "prettier/prettier": ["error", { "singleQuote": true }],
+      "import/prefer-default-export": "off"
+  
+      
+    }
+  }
```

Comment:
always leave an empty line at the end of file.

Reply:
do this for all files. Configure settings in your editor so that it always does it.